modules/lyapotaoleg/src/test/java/com/hillel/elementary/javageeks/oleg/my_labs/PolindromTest.java

```diff
@@ -0,0 +1,36 @@
+package com.hillel.elementary.javageeks.oleg.my_labs;
+
+import com.hillel.elementary.javageeks.oleg.my_labs.lab_2.Polindrom;
+import org.junit.jupiter.api.Test;
+import org.assertj.core.api.Assertions;
+
+class PolindromTest {
+    @Test
+    public void shouldResultLineSplite(){
+        String number="121,32,45,434";
+
+        String[] result={"121","32","45","434"};
+
+        Assertions.assertThat(result).isEqualTo (Polindrom.lineSplit (number));
+    }
+
+    @Test
+    public void shouldResultIsPolindromTrue(){
+        int number=11211;
+
+        Boolean result=true;
+
+        Assertions.assertThat (result).isEqualTo (Polindrom.isPolindrom (number));
```

Comment:
assertThat should hold result and isEqualTo should hold expected result, not vise versa